spring-cloud-stream-module-launcher/README.md

```diff
@@ -50,11 +55,35 @@ logsink_1    | 2015-08-11 08:25:54.909  INFO 1 --- [hannel-adapter1] o.s.cloud.s
 ```
 
 You can also run each module individually as a Docker process by passing environment variables for the module name as well as the host machine's IP address for the redis connection to be established within the container:
+To find out redis host IP:
+```
+Get the container ID of redis: `docker ps`
+Get the IP address by inspecting the container: `docker inspect <containerID>`
+
+```
+To run the modules individually on docker:
+````
+docker run -e MODULES=org.springframework.cloud.stream.module:time-source:1.0.0.BUILD-SNAPSHOT \
+ -e spring.cloud.stream.bindings.output=ticktock -e SPRING_REDIS_HOST=<Redis-Host-IP> springcloud/stream-module-launcher
 
+docker run -e MODULES=org.springframework.cloud.stream.module:log-sink:1.0.0.BUILD-SNAPSHOT \
+  -e spring.cloud.stream.bindings.input=ticktock -e SPRING_REDIS_HOST=<Redis-Host-IP> springcloud/stream-module-launcher
 ````
-docker run -p 8080:8080 -e MODULES=org.springframework.cloud.stream.module:time-source:1.0.0.BUILD-SNAPSHOT -e SPRING_REDIS_HOST=<host.ip> springcloud/stream-module-launcher
-docker run -p 8081:8081 -e MODULES=org.springframework.cloud.stream.module:log-sink:1.0.0.BUILD-SNAPSHOT -e SPRING_REDIS_HOST=<host.ip> springcloud/stream-module-launcher
+Note the binding name `ticktock` is specified for the source's output and sink's input.
+
+To run pub/sub modules individually on docker, the binding name has to start with `topic:`.
+
+````
+docker run -e MODULES=org.springframework.cloud.stream.module:time-source:1.0.0.BUILD-SNAPSHOT \
```

Comment:
should add the -p options back for these docker run commands... each can be 8080 on the container side but mapped to different ports on the docker vm, e.g. 8080:8080, 8081:8080, 8082:8080 respectively